app/src/androidTest/java/com/github/h3lp3rs/h3lp/MainPageTestActivity.kt

```diff
@@ -134,4 +147,34 @@ class MainPageTestActivity {
             clickingOnButtonWorksAndSendsIntent(FirstAidActivity::class.java, withId(R.id.button_first_aid), true)
         }
     }
+
+    @Test
+    fun getsNotifiedWhenHelpNeeded() {
+        // Mock an emergency
+        val emergencyId = 1
+        val emergenciesDb = MockDatabase()
+        val skills = HelperSkills(true, false,false,
+                                false,false, false)
+        val emergency = EmergencyInformation(emergencyId.toString(), 1.0,1.0, skills,
+                                ArrayList(listOf("Epipen")), Date(), null, ArrayList())
+        emergenciesDb.setObject(emergencyId.toString(), EmergencyInformation::class.java, emergency)
+        EMERGENCIES.db = emergenciesDb
+        val newEmergenciesDb = MockDatabase()
+        newEmergenciesDb.setInt(globalContext.getString(R.string.epipen), emergencyId)
+        NEW_EMERGENCIES.db = newEmergenciesDb
+        // Add to storage the skills
+        storageOf(SKILLS).setObject(globalContext.getString(R.string.my_skills_key),
+            HelperSkills::class.java, skills)
+        // To track notifications
+        val uiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+        launchAndDo {
+            // Should immediately receive a notification
+            uiDevice.wait(Until.hasObject(By.textStartsWith("H3LP")),3000)
+            // Get the notification box - CIRRUS DOESN'T LIKE THIS
```

Comment:
Weird that it doesn't work anymore seems to work in the notification test last sprint

Reply:
Yes exactly! But it's the combination of both! I think it's maybe because they have the same name and they are not cleared. Is it possible?